api/Screen.json

```diff
@@ -357,8 +357,7 @@
               "version_added": false
             },
             "edge": {
-              "version_added": "≤18",
-              "version_removed": "79"
+              "version_added": false
```

Comment:
Was "onmsorientationchange" really removed in Edge 12, without a replacement?

Reply:
This line's actually for the "left" property"!  Though since we know `onmsorientationchange` was supported in IE, we can infer "12" from that -- I've updated the data accordingly!

Reply:
Confirmed by http://web-confluence.appspot.com/#!/catalog?releases=%5B%22Edge_12.10240_Windows_10.0%22,%22Edge_18.17763_Windows_10.0%22%5D&q=%22onmsorientationchange%22 too